packages/admin/cms-admin/src/pages/pageTree/useCopyPastePages/sendPages.tsx

```diff
@@ -413,16 +413,17 @@ function createPageTreeNodeIdReplacements(nodes: PageClipboard[]): ReplaceDepend
 function unhandledDependenciesFromDocument(
     documentType: DocumentInterface,
     document: GQLDocument,
-    { existingReplacements }: { existingReplacements: ReplaceDependencyObject[] },
+    { existingReplacements, hasDamScope = false }: { existingReplacements: ReplaceDependencyObject[]; hasDamScope?: boolean },
```

Comment:
It seems to me that you're missing passing this option when calling the function.

Reply:
Damn, sorry 🤦‍♂️